etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -48,13 +48,14 @@ module load cray-hdf5-parallel/1.10.0.3
 # Self-Build Software #########################################################
 #
 # needs to be compiled by the user
-export BOOST_ROOT=$SCRATCH/lib/boost-1.62.0
-export ZLIB_ROOT=$SCRATCH/lib/zlib-1.2.11
-export PNG_ROOT=$SCRATCH/lib/libpng-1.6.34
-export BLOSC_ROOT=$SCRATCH/lib/blosc-1.12.1
-export PNGwriter_DIR=$SCRATCH/lib/pngwriter-0.7.0
-export ADIOS_ROOT=$SCRATCH/lib/adios-1.13.1
-export Splash_DIR=$SCRATCH/lib/splash-1.7.0
+export PICLIB="$HOME/lib"
```

Comment:
@steindev can you please call this `PIC_LIBS`?

Reply:
Done